### PR TITLE
Add dbplyr 1.3.0 release post

### PR DESCRIPTION
**Pull request by batpigandme**
This is like 99% highlight reel of the news, let me know if there's anything else I should add.

### CODE REVIEW

File: content/articles/2019-01-dbplyr-1-3-0.markdown [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+## SQL translation
+
+* The new `as.integer64(x)` is translated to `CAST(x AS BIGINT)`. 
+
+* [`cummean()`](https://dplyr.tidyverse.org/reference/cumall.html) now translates to SQL `AVG()` as opposed to `MEAN()`.
+
+* `x %in% y` is now translated to `FALSE` if `y` is empty.
+
+## Minor improvements
```
Comment (hadley) [inline content/articles/2019-01-dbplyr-1-3-0.markdown]: I think you can remove this section

File: content/articles/2019-01-dbplyr-1-3-0.markdown [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+description: > 
+  dbplyr 1.3.0 is now on CRAN.
+categories:
+  - package
+photo:
+  url: https://unsplash.com/photos/sk59I1qRfEM
+  author: Scott Webb
+---
+
+We're stoked to announce that [dbplyr](https://dbplyr.tidyverse.org) 1.3.0 is now available on CRAN. dbplyr is the database backend for dplyr, translating dplyr syntax into SQL. This is a minor release primarily for compatibility with dplyr 0.8.0. However, there have been some changes to the API, and minor improvements (the full details of which can be found in the [Changelog](https://dbplyr.tidyverse.org/news/index.html)). A more substantive update can be expected after rstudio::conf.
```
Comment (hadley) [inline content/articles/2019-01-dbplyr-1-3-0.markdown]: Do we usually say "Changelog"? (with leading capital)

Comment (batpigandme) [reply content/articles/2019-01-dbplyr-1-3-0.markdown]: No we do not. Good job, you've passed the test. 🙈

**Comment by hadley** [comment]
Thanks - I'll ping you again when it's on CRAN and we can publish

**Pull request merged**